substrafl/experiment.py

```diff
@@ -314,14 +315,23 @@ def execute_experiment(
 
     # create computation graph
     for round_idx in range(0, num_rounds + 1):
-        strategy.perform_round(
-            algo=algo,
-            train_data_nodes=train_data_nodes,
-            aggregation_node=aggregation_node,
-            additional_orgs_permissions=additional_orgs_permissions,
-            round_idx=round_idx,
-            clean_models=clean_models,
-        )
+        if round_idx == 0:
```

Comment:
Minor but I think it would be more readable to move the `if` block outside of the loop, and start the range at 1 (flat is better than nested, from the Zen of Python).